AppDashboard/templates/logs/main.html

```diff
@@ -2,14 +2,17 @@
 <!--Body content-->
 <div id="content" class="clearfix">
     <div class="contentwrapper"><!--Content wrapper-->
-
+        {% set app_page_title="Log Viewer" %}
```

Comment:
Could this be passed in instead of defined here? Using the string directly below would also be fine with me.